src/main/util/consts.py

```diff
@@ -0,0 +1,168 @@
+
+
+class ACTIVITY_TRACKER_EVENTS(Enum):
+    ACTION = 'Action'
+    COMPILATION_FINISHED = 'CompilationFinished'
+
+    @classmethod
+    def action_events(cls):
+        return ['Run', 'Rerun', 'RunClass', 'DebugClass', 'ToggleLineBreakpoint', 'Debugger.AddToWatch', 'Debug',
+                'Stop', 'Resume', 'StepInto', 'CompileDirty', 'EditorCopy', 'EditorPaste', 'EditorCut', 'ReformatCode',
+                '$Undo', '$Paste', '$Copy', 'ChooseRunConfiguration', 'CopyElement', 'PasteElement', 'CutElement']
+
+
+class DEFAULT_VALUES(Enum):
+    AGE = 0
+    EXPERIENCE = np.nan
+
+
+class TASK(Enum):
+    PIES = 'pies'
+    MAX_3 = 'max_3'
+    ZERO = 'zero'
+    ELECTION = 'election'
+    BRACKETS = 'brackets'
+    MAX_DIGIT = 'max_digit'
+
+
+class TASKS_TESTS(Enum):
+    TASKS_TESTS_PATH = ROOT_DIR + '/../../resources/tasks_tests/'
+    SOURCE_OBJECT_NAME = 'source'
+    INPUT_FILE_NAME = 'in'
+
+
+class LANGUAGE(Enum):
+    JAVA = 'java'
+    PYTHON = 'python'
+    KOTLIN = 'kotlin'
+    CPP = 'cpp'
+    NOT_DEFINED = 'not_defined'
+
+
+class TEST_RESULT(Enum):
+    INCORRECT_CODE = -1
+    LANGUAGE_NOT_DEFINED = -2
+    # It can be incorrect although, but our checking tools cannot find any errors
+    CORRECT_CODE = 0
+
+
+class FILE_SYSTEM_ITEM(Enum):
+    PATH = 0
+    SUBDIR = 1
+    FILE = 2
+
+
+EXTENSION_TO_LANGUAGE_DICT = {
+    'py': LANGUAGE.PYTHON.value,
+    'java': LANGUAGE.JAVA.value,
+    'kt': LANGUAGE.KOTLIN.value,
+    'cpp': LANGUAGE.CPP.value,
+    '': LANGUAGE.NOT_DEFINED.value
+}
+
+LANGUAGE_TO_MIN_SYMBOLS = {
+    # a = int(input())
+    # print(a)
+    LANGUAGE.PYTHON.value: 25,
+
+    # class A{
+    # public static void main(String args[]){
+    # Scanner in = new Scanner(System.in );
+    # Int a = in.nextInt();
+    # System.out.print(a);}}
+    LANGUAGE.JAVA.value: 140,
+
+    # fun main(args: Array<String>){
+    # val a:Int = readLine()!!.toInt()
+    # print(a)}
+    LANGUAGE.KOTLIN.value: 80,
+
+    # #include <iostream>
+    # int main(){
+    # int a;
+    # cin>>a;
+    # cout<<a;}
+    LANGUAGE.CPP.value: 60
+}
+
+LANGUAGE_TO_OUTPUT = {
+    LANGUAGE.PYTHON.value: "print",
+    LANGUAGE.JAVA.value: "System.out.print",
+    LANGUAGE.KOTLIN.value: "print",
+    LANGUAGE.CPP.value: "cout",
+}
+
+class SPLIT_DICT(Enum):
+    INDEX = 'index'
+    RATE = 'rate'
+    TASKS = 'tasks'
+
+
+LOGGER_FILE = ROOT_DIR + '../../../../logs.log'
+LOGGER_NAME = 'main_logger'
+
+LOGGER_TEST_FILE = ROOT_DIR + '../../../../test_logs.log'
+
+PATH_CMD_ARG = '-path'
+ENCODING = 'ISO-8859-1'
+
+DATA_FOLDER_WITH_ATI = 'ati_'
+DATA_FOLDER_WITHOUT_ATI = 'di_'
+ACTIVITY_TRACKER_FILE_NAME = 'ide-events'
+
+DATE_TIME_FORMAT = '%Y-%m-%dT%H:%M:%S.%f%z'
+MAX_DIF_SEC = 0.5
+
+TEST_DATA_PATH = ROOT_DIR + '/../../resources/test_data'
+
+PREPROCESSING_RESULT_FOLDER = 'preprocessing_result'
+
+# v 2.0 - with stopping after the first break
+RUNNING_TESTS_RESULT_FOLDER = 'running_tests_result2'
```

Comment:
Without 2 or with _: running_tests_result_2. But I think the running_tests_result is better

Reply:
Yeah, it's better, but I used the 'running_test_result' in my previous version, the one without stopping after the first false getting in tests. Since the results of these two versions are different, we don't want to write them in the same folder. Although, I can add '_' if you want

Reply:
I think it is better because we use _ for separating parts in all names in our project